modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -49,30 +50,32 @@ namespace {{packageName}}.Client
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" /> class
-        /// with default configuration and base path ({{{basePath}}}).
+        /// with default configuration and base path (<see cref="SERVER_DEFAULT_ADDRESS"/>).
         /// </summary>
-        public ApiClient()
-        {
+        public ApiClient() {
+            log.Trace("start");
```

Comment:
Don't these `start` and `end` trace messages make assumptions about how the Logger class is implemented? For example, if I were to implement logger to write message directly to a file in the format `[LEVEL] MSG`, wouldn't this just log `[TRACE] start`? If this is forcing the implementation to pull the method name off the stack, I think that would be a negative performance hit if logging is enabled.

I think it would be more helpful to embed the method name being wrapped here.